docs/framework/api-architecture.md

```diff
@@ -0,0 +1,175 @@
+# API architecture
+
+
+The term 'API' refers to that code stored in the top level api folder.
+It's important to note that only compliant code & usage will be
+supported. Usage documentation is in the [API
+v3](/api/index.md) page.
+
+## Standards {:#standards}
+
+-   The ONLY supported way to interact with any code in the API folder
+    from outside the api folder is by using the `civicrm_api()` wrapper
```

Comment:
Addressed in 6fb4575d9e9294cad36eb8bf7789e3122f0a2574 and 8510886cef61bd257b0de492e3c93006481d305f